src/OrchardCore.Modules/OrchardCore.Users/Services/DefaultExternalLoginUserToRelateFinder.cs

```diff
@@ -0,0 +1,42 @@
+
+using System.Runtime.CompilerServices;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using OrchardCore.Users.Abstractions;
+
+namespace OrchardCore.Users.Services;
+
+public class DefaultExternalLoginUserToRelateFinder : IExternalLoginUserToRelateFinder
+{
+  private readonly UserManager<IUser> _userManager;
```

Comment:
In OC we use 4 spaces not 2. Please ensure you fix these formats. Also we add a single space before a comment like 

```
// some comment
```

Reply:
fixed in bd01473